components/Alert/Alert.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 import { cva, type VariantProps } from 'class-variance-authority';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
+  'relative w-full max-w-fit rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
```

Comment:
I thought you sold me on the idea of making them all the same size? See thread here: https://github.com/LetsGetTechnical/gridiron-survivor/pull/607#discussion_r1799957376

Reply:
Agree. I'm not digging the scaled width. If an alert was 200 lines long, the alert is not extra long. Y'all figure out a good min-width.

Reply:
Can we do some research on how other websites display their notifications with color contrast? I'm not digging the black on black. Even with the colored outline, it's not that great of a contrast to signify that this is a notification.

Reply:
I added a specific maximum width to the toasts with the class `max-w-[25rem]`. I originally tried @ryandotfurrer's suggestion to use `max-w-96`, but there was a specificity issue where it would not apply, and the toast would take up the entire width of the viewport. By directly setting the maximum width of 25rem, it seemed to work.

Reply:
@kepsteen I realized why this is. Our project is currently on Tailwind 3.3.3 and I don't think `max-w-96` was added until Tailwind 3.4 🙄

The arbitrary value will have to stay right now.